demo-app/src/common/CvcField.tsx

```diff
@@ -26,20 +21,20 @@ interface CvcFieldProps extends UIComponentProps {
 }
 
 const CvcField = (props: CvcFieldProps) => {
+  const validationColours = props.isValid ? 'green' : 'red';
+  const validationColourStyle = props.isEditable ? validationColours : 'silver';
```

Comment:
We might get merchants feedback that the border does not change automatically when AccessCheckoutTextInput is disabled. Let's just and see feedback then :)